background-concealment-demo-py/BackgroundConcealmentDemo.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>It will be useful to show the image before applying the algorithm, too.. if it's possible to show two images simultaneously in the Notebook, of course.</p><p> </p><p>It will increase the pipeline complexity a bit - I mean, it should be 2 outputs, src image and out image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TolyaTalamanov/g-api-collection/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>